docs/swagger/user.yaml

```diff
@@ -465,9 +465,39 @@ paths:
         default:
           description: Successful operation
 
-#-----------------------------------------------------------------------------------------------------------------------
-# Begin Session
-#-----------------------------------------------------------------------------------------------------------------------
+  #-----------------------------------------------------------------------------------------------------------------------
+  # Begin Invite
+  #-----------------------------------------------------------------------------------------------------------------------
+  '/user/invite':
+    post:
+      tags:
+        - Invite
+      summary: Invite new users to the HumHub installation
+      description: ''
+      produces:
+        - application/json
+        - application/xml
+      parameters:
+        - in: body
+          name: body
+          description: Email of people you want to invite
+          required: true
+          schema:
+            type: object
+            required:
+              - email
+            properties:
+              email:
+                type: string
```

Comment:
Can we use an array here? This probably makes automated things easier .

Reply:
Yes, array is better. I had initially chosen the comma separated string to use the Invite model.
Changed in commit https://github.com/humhub/rest/pull/138/commits/a6dc8a30d4d02691daedfa64d3ef3488b33af381